### PR TITLE
fix(secrets): value should be enclosed in quotes

### PR DESCRIPTION
**Pull request by waaronmorris**
[Resolving Issue 420](https://github.com/duckdb/dbt-duckdb/issues/420)

### CODE REVIEW

**Comment by jwills** [comment]
ack, thank you @waaronmorris! cc @guenp

**Comment by guenp** [comment]
Thanks @waaronmorris, this is a great catch!
Could you also update the [unit tests](https://github.com/duckdb/dbt-duckdb/blob/ec65e430bf387235924805a75585dda6ce3ede67/tests/unit/test_credentials.py#L39)?

File: dbt/adapters/duckdb/secrets.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def to_sql(self) -> str:
         params.update(params.pop("secret_kwargs", {}))
         params_sql = f",\n{tab}".join(
             [
-                f"{key} {value}"
+                f"{key} '{value}'"
```
Comment (guenp) [inline dbt/adapters/duckdb/secrets.py:50]:
```suggestion
                f"{key} '{value}'" if key not in ["type", "provider"] else f"{key} {value}"
```

**Review by guenp — CHANGES_REQUESTED** [review]
We also need to add quotes to all the unit tests - see https://github.com/duckdb/dbt-duckdb/commit/3190554bcd926946e7e548e4be4828d64b6cfe55

**Pull request merged**